store/tikv/2pc.go

```diff
@@ -224,9 +224,19 @@ func (c *twoPhaseCommitter) doActionOnKeys(bo *Backoffer, action twoPhaseCommitA
 	if action == actionCommit {
 		// Commit secondary batches in background goroutine to reduce latency.
 		twoPhaseCommitGP.Go(func() {
+			start := time.Now()
+			secondaryLockCleanupWorkerGauge.Inc()
+			defer func() {
+				secondaryLockCleanupWorkerGauge.Dec()
+				secondaryLockCleanupHistogram.WithLabelValues("commit_cleanup_locks").Observe(time.Since(start).Seconds())
```

Comment:
using `commit` as the tag name is enough.

Reply:
ok